aggsender/validator/local_validator.go

```diff
@@ -82,7 +90,7 @@ func (a *LocalValidator) ValidateAndSignCertificate(
 	a.log.Infof("certificate validation passed: %s", certificate.Brief())
 
 	// local validator does not sign the certificate, it just validates it
-	return nil, nil
+	return make([]byte, aggkitcommon.SignatureSize), nil
```

Comment:
Why it is not ok to return a `nil`? Is it because of the signature size validation introduced?

Can it be that the remote validator returns a nil in some occasions (I have tried to follow that execution path, but got lost in a meantime)...

Maybe it would make sense to expose a global var called `EmptySignature` and use it instead?

Reply:
You are calling both local and remote validators in the same `pollValidators` function. Since we introduce a metric about invalid signature, for local validator, this would always fail.